functions.php

```diff
@@ -6,14 +6,39 @@
 
 
 add_action( 'wp_enqueue_scripts', 'enqueue_child_styles', 99);
+function theme_filectime($path) {
+	return filectime(get_stylesheet_directory() . '/dist/bundle.js');
+}
 function enqueue_child_styles() {
-    $css_creation = filectime(get_stylesheet_directory() . '/dist/bundle.js');
-
-    wp_enqueue_script( 'validator', get_stylesheet_directory_uri() . '/vendor/validator.js', array('jquery'), $css_creation, true );
-    wp_enqueue_script( 'mask', get_stylesheet_directory_uri() . '/vendor/jquery.mask.min.js', array('jquery'), $css_creation, true );
-    wp_enqueue_script( 'bundle', get_stylesheet_directory_uri() . '/dist/bundle.js', array('jquery','validator'), $css_creation, true );
-    wp_localize_script( 'bundle', 'adminAjaxUrl', admin_url( 'admin-ajax.php' ) );
-    wp_localize_script( 'bundle', 'lang', ICL_LANGUAGE_CODE);
+	wp_enqueue_style(
+		'gplux_style',
+		get_stylesheet_directory_uri() . 'style.css',
+		[],
+		theme_filectime( 'style.css' )
+	);
+	wp_enqueue_script(
+		'validator',
+		get_stylesheet_directory_uri() . '/vendor/validator.js',
+		array( 'jquery' ),
+		theme_filectime('/vendor/validator.js'),
```

Comment:
I changed these to look at the right file for modifications. Otherwise if any of these are updated without updating `/dist/bundle.js`, the URL would stay the same and the old version would still come from Cloudflare cache for a long time.